$OpenBSD$

Use unsigned arithmetic to prevent signed overflow error from gcc 6.
error: shift expression '(1048575 << 20)' overflows [-fpermissive]
error: enumerator value for 'BackgroundMask' is not an integer constant

BackgroundMask overflows because it sets bits 20 to 39, but there are
no bits above 31 in an int.

Index: src/xmlpatterns/api/qcoloroutput_p.h
--- src/xmlpatterns/api/qcoloroutput_p.h.orig
+++ src/xmlpatterns/api/qcoloroutput_p.h
@@ -71,7 +71,7 @@ namespace QPatternist
             BackgroundShift = 20,
             SpecialShift    = 20,
             ForegroundMask  = ((1 << ForegroundShift) - 1) << ForegroundShift,
-            BackgroundMask  = ((1 << BackgroundShift) - 1) << BackgroundShift
+            BackgroundMask  = ((1U << BackgroundShift) - 1) << BackgroundShift
         };
 
     public:
